class-pwa-manifest.php

```diff
@@ -35,14 +35,14 @@ public function filter_jetpack_pwa_manifest( $manifest ) {
 		$icon_id = get_theme_mod( 'rpe_pwa_icon' );
 		// Get the site icon id if PWA icon is not set.
 		$icon_id = ! empty( $icon_id ) ? $icon_id : get_option( 'site_icon' );
-		$icons = array();
+		$icons   = array();
```

Comment:
We should use `[]` array syntax instead of `array()`.

Reply:
You can replace it everywhere.

Reply:
Changes added :+1: